### ISSUE TITLE
Flaky Mac module_test_ios now failing regularly

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#88495]
Following the Engine roll here https://github.com/flutter/flutter/pull/88481, several tests in "Mac module_test_ios", which is marked flaky, are now failing every time. That Engine roll contained only a Fuchsia SDK roll, which is unrelated, and the change here https://github.com/flutter/engine/pull/27893.

Example failure here https://ci.chromium.org/ui/p/flutter/builders/prod/Mac%20module_test_ios/3861/overview

**Comment by jmagman** [comment]
From https://github.com/flutter/flutter/issues/88547:

> ```
> Crashed Thread:        0  Dispatch queue: com.apple.main-thread
> 
> Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
> Exception Codes:       EXC_I386_GPFLT
> Exception Note:        EXC_CORPSE_NOTIFY
> 
> Termination Signal:    Segmentation fault: 11
> Termination Reason:    Namespace SIGNAL, Code 0xb
> Terminating Process:   exc handler [41758]
> 
> Application Specific Information:
> objc_msgSend() selector name: length
> CoreSimulator 732.18.6 - Device: TestAdd2AppSim (C62405C6-D622-424B-A0DA-E483AE666793) - Runtime: iOS 14.3 (18C61) - DeviceType: iPhone 11
> 
> Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
> 0   libobjc.A.dylib               	0x000000010940478b objc_msgSend + 11
> 1   io.flutter.flutter            	0x0000000106102e48 -[SemanticsObject accessibilityAttributedLabel] + 52
> 2   com.apple.UIAccessibility     	0x000000013741eedf -[NSObject(AXPrivCategory) _accessibilityPotentiallyAttributedValueForNonAttributedSelector:attributedSelector:] + 237
> ```
> 
> [crash.txt](https://github.com/flutter/flutter/files/7017642/crash.txt)
> 
> Plugins tree is closed:
> https://cirrus-ci.com/task/5163527850688512?logs=native_test#L415-L418
> 
> `module_test_ios` on flutter/flutter also failed. It's unfortunately marked flaky, so it didn't close the tree.
> https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8838512374717310545/+/u/run_module_test_ios/test_stdout
> 
> Started on engine roll #88481, assuming [flutter/engine#27893](https://github.com/flutter/engine/pull/27893).

**Comment by zanderso** [comment]
Revert is landed.
